src/main/scala/uk/gov/hmrc/mongo/ReactiveRepository.scala

```diff
@@ -99,13 +99,11 @@ abstract class ReactiveRepository[A <: Any, ID <: Any](collectionName: String,
   private def ensureIndex(index: Index)(implicit ec: ExecutionContext): Future[Boolean] = {
     collection.indexesManager.create(index).map(wr => {
      if(!wr.ok) {
-       val maybeMsg = for {
-         msg <- wr.errmsg
-         m <- if (msg.contains(DuplicateKeyError)) {
+       val msg = wr.writeErrors.mkString(", ")
```

Comment:
`writeErrors` is now a sequence of strings. This means that we create a big string from the sequence to keep compatibility. Also I hate this method because it brings in the global execution context. Due to compatibility, I have not fixed the global `ExecutionContext` import yet. That is something for the future.